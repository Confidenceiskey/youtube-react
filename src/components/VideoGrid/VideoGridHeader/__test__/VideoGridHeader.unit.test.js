import React from 'react';
import { shallow } from 'enzyme';
import VideoGridHeader from '../VideoGridHeader';

describe('', () => {
  it('renders VideoGridHeader w/o props', () => {
    const wrapper = shallow(
      <VideoGridHeader />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('renders VideoGridHeader w/ title prop', () => {
    const wrapper = shallow(
      <VideoGridHeader title='Trending' />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('renders VideoGridHeader w/ blank title prop', () => {
    const wrapper = shallow(
      <VideoGridHeader title='' />
    );
    expect(wrapper).toMatchSnapshot();
  });
});