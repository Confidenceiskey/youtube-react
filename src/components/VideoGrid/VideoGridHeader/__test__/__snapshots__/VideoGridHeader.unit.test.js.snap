// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders VideoGridHeader w/ blank title prop 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;

exports[`renders VideoGridHeader w/ title prop 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  >
    Trending
  </span>
</div>
`;

exports[`renders VideoGridHeader w/o props 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;
